packages/fuel-gauge/src/contract.test.ts

```diff
@@ -713,7 +713,7 @@ describe('Contract', () => {
     const initialBalance = new BN(
       await contract.getBalance(await provider.getBaseAssetId())
     ).toNumber();
-    const amountToContract = bn(2).pow(62); // Very big number
+    const amountToContract = bn(2).pow(61); // Very big number
```

Comment:
What's up with this change?

Reply:
I made this change to make the test pass. This test attempted to transfer the whole funding amount to a contract, which is expected to not work since no amount will be left to pay the TX fee.

It’s unclear why the test was working previously when using BN. Perhaps it’s related to how very large numbers in JavaScript are handled, as they might be implicitly modified."

Reply:
Got it. There's something wrong with the browser tests, tho.